src/cgnstools/tkogl/tkogl.c

```diff
@@ -22,6 +22,12 @@
 #include "printstr.h"
 #include "feedback.h"
 
+#if ! defined(__WIN32__) && ! defined(_WIN32)
+/* For TkWmAddToColormapWindows. */
+#define _TKPORT /* Typical installations cannot find tkPort.h. */
+#include <tk-private/generic/tkInt.h>
```

Comment:
This breaks opensuse builds, where tkInt.h is in /usr/include

Reply:
We'll have to see how Kitware is handling it:

https://github.com/Kitware/VTK/blob/dbd25545a57c976b15a95f4651df17bc84540f41/Rendering/Tk/vtkTkRenderWidget.cxx#L21

I did a little searching for Autotools, but definitely no great solutions for finding it other than using pkg-config or requiring the TCL/TK_INCLUDE_DIR variable, as I don't think providing TKdir is enough to find it. Or cycle through a list of common directories where it is found.